shared/js/background/chrome-wrapper.es6.js

```diff
@@ -33,21 +33,19 @@ let mergeSavedSettings = (settings, results) => {
     return Object.assign(settings, results)
 }
 
-let injectATBScripts = () => {
-    chrome.tabs.query({ url: 'https://*.duckduckgo.com/*' }, (tabs) => {
-        let i = tabs.length
-        let tab
+let getDDGTabUrls = () => {
+    return new Promise((resolve) => {
+        chrome.tabs.query({ url: 'https://*.duckduckgo.com/*' }, (tabs) => {
+            tabs = tabs || []
 
-        while (i--) {
-            tab = tabs[i]
-
-            chrome.tabs.executeScript(tab.id, {
-                file: '/public/js/content-scripts/on-install.js'
-            })
-            chrome.tabs.insertCSS(tab.id, {
-                file: '/public/css/noatb.css'
+            tabs.forEach(tab => {
+                chrome.tabs.insertCSS(tab.id, {
+                    file: '/public/css/noatb.css'
+                })
             })
-        }
+
+            resolve(tabs.map(tab => tab.url))
```

Comment:
I'm just passing the URLs from here, then reading them back from within the ATB module. That way it's easier to add tests.